server/notification-providers/telegram.js

```diff
@@ -30,9 +30,9 @@ class Telegram extends NotificationProvider {
                 }
             }
 
-            await axios.get(`${url}/bot${notification.telegramBotToken}/sendMessage`, {
-                params: params,
-            });
+            let axiosConfig = this.getAxiosConfigWithProxy({ params });
+
+            await axios.get(`${url}/bot${notification.telegramBotToken}/sendMessage`, axiosConfig);
```

Comment:
Let's roll out this change across all notifications. This should be easy enough and I don't see how this can cause harm.

I don't think this needs testing if it works on one notification provider..